src/Agent.Listener/CommandSettings.cs

```diff
@@ -163,6 +164,22 @@ public bool GetRunAsService()
                 defaultValue: false);
         }
 
+        public bool GetEnableAutoLogon()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.EnableAutoLogon,
+                description: StringUtil.Loc("EnableAutoLogon"),
+                defaultValue: false);
+        }
+
+        public bool GetRestartNow()
```

Comment:
it might be worth having a conversation whether true is the correct default since we basically can't change the default later.